pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreatorTest.java

```diff
@@ -250,6 +253,16 @@ public void testAddMinMaxValue() {
         stringMaxValue, FieldSpec.DataType.STRING);
     compareLongValuesWithColumnMinMax(stringMinValue, stringMaxValue, props, FieldSpec.DataType.STRING);
 
+    // test for value length grater than METADATA_PROPERTY_LENGTH_LIMIT with random string having ascii characters.
```

Comment:
Added ASCII-based random string to test the scenario if the original values have characters which need escape.